docs/configuration.md

```diff
@@ -51,6 +51,11 @@ export CLOUDBREAK_SMTP_SENDER_PORT=
 export CLOUDBREAK_SMTP_SENDER_FROM=
 ```
 
+## Cloudbreak development
+
+To make local development easier we introduced some environment configuration.
+- Set local Uluwatu source location with ULUWATU_VOLUME_HOST variable
+
```

Comment:
adda note about. Maybe a new **Hacking on Uluwatu** section can be added with extra info:

> Please note that you need to `generate` you config files. So the usual process is:

```
# edit Profile by adding ULUWATU_VOLUME_HOST
cbd regenerate
cbd kill
cbd start
```

- If you make changes to html/css/client side js you just refresh the browser.
- If you make changes in server side js, you have to restart the container

```
docker restart cbreak_uluwatu_1
```